Scripts/Utils.ps1

```diff
@@ -0,0 +1,39 @@
+﻿function CleanTempFolder()
+{
+    if (Test-Path $env:temp)
+    {
+        Write-Host 'Deleting temp folder'
+        rmdir $env:temp -Recurse -ErrorAction SilentlyContinue
```

Comment:
Not sure if this is a good idea to wipe out global temp folder. Other running processes may use it.